cmd/podman/machine/list.go

```diff
@@ -202,8 +203,8 @@ func toHumanFormat(vms []*machine.ListResponse, defaultCon string) []*entities.L
 		response.Created = units.HumanDuration(time.Since(vm.CreatedAt)) + " ago"
 		response.VMType = vm.VMType
 		response.CPUs = vm.CPUs
-		response.Memory = units.BytesSize(float64(vm.Memory))
-		response.DiskSize = units.BytesSize(float64(vm.DiskSize))
+		response.Memory = units.BytesSize(float64(strongunits.MiB(vm.Memory).ToBytes()))
```

Comment:
I think the non-human response (above) has the same issue, so make sure to change both of them (i.e. all four)

```json
        "CPUs": 4,
        "Memory": "2147483648",
        "DiskSize": "107374182400",

```

Reply:
Uh, I understand.

in v4
```
% podman machine list
NAME                     VM TYPE     CREATED        LAST UP     CPUS        MEMORY      DISK SIZE
podman-machine-default*  applehv     2 minutes ago  Never       5           2GiB        100GiB
% podman machine list --format json
...
        "CPUs": 5,
        "Memory": "2147483648",
        "DiskSize": "107374182400",
...
```

in v5(no-fix)
```
% ./bin/darwin/podman machine list
NAME                     VM TYPE     CREATED      LAST UP     CPUS        MEMORY      DISK SIZE
podman-machine-default*  applehv     6 hours ago  Never       5           2KiB        100B
% ./bin/darwin/podman machine list --format json
...
        "CPUs": 5,
        "Memory": "2048",
        "DiskSize": "100",
...
```

So I need to make "func toMachineFormat" look like this, right?
```
		response.Memory = strUint(uint64(strongunits.MiB(vm.Memory).ToBytes()))
		response.DiskSize = strUint(uint64(strongunits.GiB(vm.DiskSize).ToBytes()))
```

Reply:
minor detail: I think you have an extra `uint64` cast in there, since ToBytes already returns `uint64`?

Reply:
Added the following error if uint64 is omitted.
```
cannot use strongunits.MiB(vm.Memory).ToBytes() (value of type strongunits.B) as uint64 value in argument to strUint
```

Reply:
Thanks, missed that it added some weird subtype